definition/latitude_longitude_properties.md

```diff
@@ -0,0 +1,65 @@
+## Various latitude/longitude properties
+
+### geom
+
+Find out more about the `geom` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/geom).
+
+The `geom:latitude` and `geom:longitude` represent a geometric centroid for a record.
+
+These values are machine-generated math centroids for a record's geometry.
+
+### intersection
+
+Find out more about the `intersection` property [here](https://github.com/whosonfirst/whosonfirst-properties/tree/master/properties/intersection).
+
+The `intersection:latitude` and `intersection:longitude` are used in [intersection records](https://whosonfirst.mapzen.com/spelunker/placetypes/intersection/) to indicate the location of an intersection. 
```

Comment:
We should note:

At the time of this writing intersection centroids are only available on features of `intersection` placetype.